requirements/edx/base.in

```diff
@@ -100,6 +100,7 @@ help-tokens
 html5lib                            # HTML parser, used for capa problems
 ipaddress                           # Ip network support for Embargo feature
 jsonfield                           # Django model field for validated JSON; used in several apps
+laboratory                          # Library for testing that code refactors/infrastructure changes produce identical results
```

Comment:
if we use this in prod could we perhaps demo this in sharing time (and/or demo/message to broader engineering org)

Reply:
Yup, we should probably do that.